plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -17,7 +17,8 @@ class DeployExecutor
     def initialize(job, output)
       @output = output
       @job = job
-      @reference = job.deploy.reference
+      @deploy = job.deploy
```

Comment:
either use `@job.deploy` or use `@deploy` everywhere for consistency

Reply:
ah didn't realise how many places it was used.